examples/with-one-login-many-subdomains/frontend/src/App.js

```diff
@@ -18,6 +18,11 @@ SuperTokens.init({
         websiteBasePath: "/",
     },
     usesDynamicLoginMethods: true,
+    getRedirectionURL: async (context) => {
+        if (context.action === "TO_AUTH") {
+            return "/auth";
+        }
+    },
     recipeList: [
```

Comment:
what happens if the user navigates to one of the sub domains and is not logged in?